build.sbt

```diff
@@ -24,9 +25,13 @@ organization := "nl.rhinofly"
 
 publishTo := rhinoflyRepo(version.value)
 
+fork in Test := true
+
+javaOptions in Test += "-Dconfig.file=src/test/conf/application.conf"
+
 def rhinoflyRepo(version: String) = {
     val repo = if (version endsWith "SNAPSHOT") "snapshot" else "release"
     Some("Rhinofly Internal " + repo.capitalize + " Repository" at "http://maven-repository.rhinofly.net:8081/artifactory/libs-" + repo + "-local")
 }
 
-credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
\ No newline at end of file
+credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
```

Comment:
Volgens mij hebben sommige van ons dit geconfigureerd staan in hun globale SBT instellingen. Misschien dat we met zijn alle afspreken dat we het globaal definiëren waardoor dit niet langer nodig is.